src/transporters.ts

```diff
@@ -69,17 +72,17 @@ export default class DefaultTransporter {
       // responds without proper content-type.
       try {
         body = JSON.parse(body);
-      } catch (err) { /* no op */ }
+      } catch (err) { /* no op */
+      }
 
       if (body && body.error && res.statusCode !== 200) {
         if (typeof body.error === 'string') {
           err = new Error(body.error);
           err.code = res.statusCode;
 
         } else if (Array.isArray(body.error.errors)) {
-          err = new Error(body.error.errors.map(
-                          (err2) => err2.message
-                        ).join('\n'));
+          err = new Error(
+              body.error.errors.map((err2) => err2.message).join('\n'));
```

Comment:
Do we have an opinion in the style guide on using no parens with single argument lambdas?